airflow/gcp/hooks/gcs.py

```diff
@@ -323,6 +323,26 @@ def delete(self, bucket_name, object_name):
 
         self.log.info('Blob %s deleted.', object_name)
 
+    def delete_bucket(self, bucket_name: str):
+        """
+        Delete an bucket object from the Google Cloud Storage
+
+        :param bucket_name: name of the bucket which will be deleted
+        :type bucket_name: str
+        """
+
+        client = storage.Client()
+        bucket = client.bucket(bucket_name)
+
+        if not bucket.exists():
+            self.log.warning("Bucket {} not exists".format(bucket_name))
```

Comment:
```suggestion
            self.log.info("Bucket {} not exists".format(bucket_name))
```

Reply:
Should not be `warning`? Why `info`? 
I mean, user should be warn about non existing bucket, what do you think?

Reply:
We had offline talk about airflow idempotency and this case. We do not want to use `warning` because in this case the condition is met.